src/locales/fr/dialogue.ts

```diff
@@ -36,26 +36,26 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "lass": {
     "encounter": {
-      1: "Let's have a battle, shall we?",
-      2: "You look like a new trainer. Let's have a battle!",
-      3: "I don't recognize you. How about a battle?",
-      4: "Let's have a fun Pokémon battle!",
-      5: "I'll show you the ropes of how to really use Pokémon!",
-      6: "A serious battle starts from a serious beginning! Are you sure you're ready?",
-      7: "You're only young once. And you only get one shot at a given battle. Soon, you'll be nothing but a memory.",
-      8: "You'd better go easy on me, OK? Though I'll be seriously fighting!",
-      9: "School is boring. I've got nothing to do. Yawn. I'm only battling to kill the time."
-    },
-    "victory": {
-      1: "That was impressive! I've got a lot to learn.",
-      2: "I didn't think you'd beat me that bad…",
-      3: "I hope we get to have a rematch some day.",
-      4: "That was pretty amazingly fun! You've totally exhausted me…",
-      5: "You actually taught me a lesson! You're pretty amazing!",
-      6: "Seriously, I lost. That is, like, seriously depressing, but you were seriously cool.",
-      7: "I don't need memories like this. Deleting memory…",
-      8: "Hey! I told you to go easy on me! Still, you're pretty cool when you're serious.",
-      9: "I'm actually getting tired of battling… There's gotta be something new to do…"
+      1: "Affrontons-nous, d'accord ?",
+      2: "Tu as l'air d'un nouveau dresseur. Battons nous !",
+      3: "Je ne te reconnais pas. Que dirais-tu de te battre ?",
+      4: "Amusons-nous pendant ce combat Pokémon !",
+      5: "Je vais t'apprendre à te battre avec des Pokémon !",
+      6: "Un combat doit être pris au sérieux. Es-tu prêt à te battre ?",
+      7: "Tu ne seras pas jeune éternellement. Tu n'auras qu'une chance pendant un combat. Bientôt, tu ne seras plus qu'un souvenir.",
+      8: "Tu ferais mieux d'y aller doucement avec moi. Mais je vais me battre sérieusement !",
+      9: "Je m'ennuie à l'école. Je n'ai rien a faire. *Baille*. Je me bats juste pour passer le temps."
```

Comment:
```suggestion
      9: "Je m'ennuie à l'école. Je n'ai rien à faire. *Baille*. Je me bats juste pour passer le temps."
```

![image](https://github.com/pagefaultgames/pokerogue/assets/8146474/cfdd1258-743a-4cfd-93fe-a03527f252cf)